### ISSUE TITLE
Max retries exceeded

### ISSUE THREAD

**Issue by sunnysaxena** [TheLastBen/fast-stable-diffusion#1778]
getting errors while calling txt2img api

requests.exceptions.ConnectionError: HTTPSConnectionPool(host="%3cmodule%20'app.url'%20from%20'", port=443): Max retries exceeded with url: /home/sunny/PycharmProjects/pythonStableDiffusion/app/url.py'%3E:443/sdapi/v1/txt2img (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff5b8fb7640>: Failed to establish a new connection: [Errno -2] Name or service not known'))

**Comment by TheLastBen** [comment]
for that type of errors, you should check in this repo https://github.com/AUTOMATIC1111/stable-diffusion-webui

**Comment by sunnysaxena** [comment]
Apart from this how do I get the static Public URL?

Model loaded in 52.5s (load weights from disk: 46.9s, create model: 1.0s, apply weights to model: 2.6s, apply half(): 1.3s, move model to device: 0.6s).
**Running on public URL: https://1467d1a6-5e43-4a2442.gradio.live**

**Comment by TheLastBen** [comment]
> Running on public URL: https://1467d1a6-5e43-4a2442.gradio.live

this is the public url

**Comment by sunnysaxena** [comment]
Right this is the public URL, but it changes every time, I need the static URL

**Comment by TheLastBen** [comment]
I don't think you can do that with colab, you can do it with RunPod https://www.runpod.io/console/gpu-secure-cloud?template=runpod-stable-unified

**Comment by sunnysaxena** [comment]
Which one should I use, could you please suggest me?

**Comment by TheLastBen** [comment]
choose the template fast stable diffusion, inside the workspace, there are three notebooks, A1111, DBv1 and DBv2

**Comment by sunnysaxena** [comment]
please do me one more favor, could you please tell me which one should I use GPU instance?
